data/json/itemgroups/clothing.json

```diff
@@ -188,5 +188,15 @@
       [ "pocketwatch", 25 ],
       [ "wristwatch", 100 ]
     ]
+  },
```

Comment:
Inconsistent with the system used in this file (see the descriptions) and therefore of limited reuse.

Start with `clothing_work_pants`, `clothing_work_hat` etc and then define `clothing_work_set` using these. These groups are much more reusable whilst still serving your purposes. You can define any uniquely required combinations within your mapgen files (eg. `clothing_work_hat`, `clotyhing_work_mask`, `tools_lighting_industrial`).

Once you've defined `clothing_work_set` you can use these to define `monsterdrops/zombie_Technician.json` following the pattern used by for example the survivor zombie